js/clinicalTimeline.js

```diff
@@ -27,10 +27,12 @@ var clinicalTimeline = (function(){
     endAllowedOverview = 0,
     overviewX = margin.overviewAxis.left,
     chart=null,
-    timelineBool = {};
+    timelineBool = {},
+    clinicalTimelinePlugins;
   timelineBool.enableTrackTooltips = true,
   timelineBool.enableZoom = true,
   timelineBool.advancedView=false,
+  timelineBool.enableOverviewAxis = true,
```

Comment:
Is there someway we can get rid of the bools? All these seem to be related to plugins, not `clinicalTimeline.js`. I don't think an enable/disable boolean is necessary if you supply the whole plugin class/object itself.